cypress/support/commands.js

```diff
@@ -64,4 +87,84 @@ Cypress.Commands.add('createFleet', () => {
     cy.get('#rich-validation-field-name').type('test-fleet')
     cy.get('#rich-validation-field-name').should('have.value', 'test-fleet')
     cy.get('.pf-v5-l-stack__item > .pf-v5-c-label-group > .pf-v5-c-label-group__main > .pf-v5-c-label-group__list').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('#textfield-osImage').should('be.visible')
+    cy.get('#textfield-osImage').type(img)
+    cy.get('#textfield-osImage').should('have.value', img)
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-title').should('contain', 'test-fleet')
+    cy.get('.pf-v5-c-description-list__text > .pf-v5-l-flex', { timeout: 50000 }).should('contain', 'Valid')
+})
+Cypress.Commands.add('editFleet', () => {
+    cy.get('#nav-toggle').click()
+    cy.contains('Edge Management').click()
+    cy.contains('Fleets').click()
+    cy.get('[data-label="Name"]').contains('test-fleet')
+    cy.get('.pf-v5-c-table__action > .pf-v5-c-menu-toggle').click()
+    cy.get('[data-label="Name"] > .fctl-resource-link > a > .fctl-resource-link__text').contains('test-fleet')
+    cy.get('.pf-v5-c-table__action > .pf-v5-c-menu-toggle').should('be.visible')
+    cy.get('.pf-v5-c-table__action > .pf-v5-c-menu-toggle').click()
+    cy.get('[data-ouia-component-id="OUIA-Generated-DropdownItem-8"] > .pf-v5-c-menu__item > .pf-v5-c-menu__item-main > .pf-v5-c-menu__item-text').click()
+    cy.get(':nth-child(2) > .pf-v5-c-form__group-label > .pf-v5-c-form__label > .pf-v5-c-form__label-text').should('contain', 'Fleet name')
+    cy.get('.pf-v5-c-form__group-control > .pf-v5-c-label-group > .pf-v5-c-label-group__main > .pf-v5-c-label-group__list > .pf-v5-c-label-group__list-item > .pf-v5-c-button').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-title').should('contain', 'test-fleet')
+})
+Cypress.Commands.add('deleteFleet', () => {
+    cy.get('#nav-toggle').click()
+    cy.contains('Edge Management').click()
+    cy.contains('Fleets').click()
+    cy.get('[data-label="Name"]').contains('test-fleet')
+    cy.get('.pf-v5-c-table__tbody > .pf-v5-c-table__tr > .pf-v5-c-table__check > label > input').click()
+        cy.get('#pf-random-id-63 > :nth-child(1) > .pf-v5-c-toolbar__content-section > :nth-child(3) > .pf-v5-c-button').click()
```

Comment:
`random-id-X`  are created by Patternfly and the numbering may change on every render so they cannot be used.

```suggestion
        cy.get('#pf-random-id-63 > :nth-child(1) > .pf-v5-c-toolbar__content-section > :nth-child(3) > .pf-v5-c-button').click()
```